app/views/layouts/_matomo_tag.html.haml

```diff
@@ -1,15 +1,24 @@
+
+- if Spree::Config.matomo_tag_manager_url.present?
+  :javascript
+    var _mtm = _mtm || [];
+    _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    var u="#{Spree::Config.matomo_tag_manager_url}";
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u; s.parentNode.insertBefore(g,s);
+
```

Comment:
This new setting, matomo_tag_manager_url, will never work without matomo_url being set, right?
I'd put this code and conditional **below** the basic matomo tag/conditional.

If matomo_url is not set and the tag_manager is set, I wonder if it doesnt blow up the js code on the browser. In that case I'd put this matomo_tag_manager_url code/condition **inside** the matomo_url conditional.

Reply:
Hi @luisramos0 after further investigation I changed the approach to allowing both scripts at the same time - covered in the PR description:

> Originally I thought that it would be best to have the Matomo Tag Manager setting override the existing Matomo Analytics setting. However I have found in testing that a combination of the two works more reliably. Therefore I have changed this so that both embed codes can be added if needed.